content/francais/python-turtle/activity-10.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Activité 10 - Randomiser la couleur des pétales de votre fleur mandala"
+date: 2024-09-17T13:24:17-07:00
+weight: 13
+draft: false
+---
+
+![alt text width="70%"](../media/mandala-color.png "mandala fleur avec pétales de couleur aléatoire")
+
+Nous avons réussi à définir la couleur de la fleur mandala sur une valeur RVB. Maintenant, explorons le changement de couleur des pétales individuels de la fleur mandala ! 
```

Comment:
Got changed for "RVB" in french form (Rouge, Vert, Blue) instead of RGB. Not sure if RVB is standard translation for RGB in french. 
In spanish I'm aware of RGB and its english acronym.

Reply:
I didnt know there is a style RVB similar to RGB, let me keep it to RGB and see if anyone has any suggestions.